### PR TITLE
Created my first Goodie

### PR DESCRIPTION
**Pull request by kleinjoshuaa**
# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**
Tells anyone that searches for my name that I'm awesome!

**What problem does your Instant Answer solve (Why is it better than organic links)?**
Teaches me how to write a duckduckgo goodie

**What is the data source for your Instant Answer? (Provide a link if possible)**
N/A

**Why did you choose this data source?**
N/A

**Are there any other alternative (better) data sources?**
N/A

**What are some example queries that trigger this Instant Answer?**
duckduckhack kleinjoshuaa

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
me, mainly

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
The [quickstart](https://duck.co/duckduckhack/goodie_quickstart)

**Which existing Instant Answers will this one supercede/overlap with?**
Other isAwesome answers

**Are you having any problems? Do you need our help with anything?**
N/A

**Where did you hear about DuckDuckHack? (For first time contributors)**
duckduckgo

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
goodie

## Checklist
Please place an 'X' where appropriate.

```
- [x] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [x] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [x] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [x] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```

### CODE REVIEW

File: lib/DDG/Goodie/IsAwesome/kleinjoshuaa.pm [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package DDG::Goodie::IsAwesome::kleinjoshuaa;
+# ABSTRACT: kleinjoshuaa's first goodie
+
+
```
Comment (mintsoft) [inline lib/DDG/Goodie/IsAwesome/kleinjoshuaa.pm]: Can you add `use strict;` here? It doesn't make any functional difference, however it forces Github to see it as Perl5

Comment (kleinjoshuaa) [inline lib/DDG/Goodie/IsAwesome/kleinjoshuaa.pm]: Ok, should be updated now

**Comment by mintsoft** [comment]
Excellent! @kleinjoshuaa Congratulations on your first goodie :tada: 

We'd love you to stick around and contribute some more, there're always some suggestions on the ideas forum (https://duck.co/ideas) or the github issues (https://github.com/duckduckgo/zeroclickinfo-goodies/labels/Low-Hanging%20Fruit)

**Comment by moollaza** [comment]
@kleinjoshuaa congrats! It's live: https://duckduckgo.com/?q=duckduckhack+kleinjoshuaa&ia=answer

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's & Issues or even submit more Instant Answers!

**Pull request merged**